projects/plugins/boost/app/assets/src/js/react-components/sections/header/index.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import React from 'react';
+import { navigate } from '../../../utils/navigate';
+import BackButton from '../../components/back-button';
+import ChevronRight from '../../svg/chevron-right';
+import Logo from '../../svg/logo';
+import styles from './styles.module.scss';
+
+interface HeaderProps {
+	subPageTitle?: string;
+	children?: React.ReactNode;
+}
+
+const Header = ( { subPageTitle = '', children }: HeaderProps ) => {
+	return (
+		<div className={ classNames( styles[ 'jb-dashboard-header' ] ) }>
+			<div className={ classNames( 'jb-container', styles.masthead ) }>
+				<div className={ classNames( styles[ 'nav-area' ] ) }>
+					<div
+						className={ classNames( styles.logo ) }
+						onClick={ () => navigate( '/' ) }
+						onKeyDown={ event => {
+							if ( event.key === 'Enter' || event.key === ' ' ) {
+								navigate( '/' );
+							}
+						} }
+						role="button"
+						tabIndex={ 0 }
+					>
+						<Logo />
+					</div>
+
+					{ subPageTitle !== '' && (
+						<>
+							<div className={ classNames( styles.chevron ) }>
+								<ChevronRight />
+							</div>
+							<div className={ classNames( styles.subpage ) }>{ subPageTitle }</div>
+						</>
+					) }
+				</div>
+
+				{ children }
+			</div>
+
+			{ subPageTitle !== '' && (
```

Comment:
Just an observation - maybe a separate prop for this can be used? That way, we can show the back button without having a sub title.

Reply:
I think we should add the prop when we need to.